src/broker/cas_execute.c

```diff
@@ -11753,4 +11765,36 @@ ux_cgw_free_stmt (T_SRV_HANDLE * srv_handle)
 {
   cgw_free_stmt (srv_handle);
 }
+
+static char
+ux_cgw_get_stmt_type (char *stmt)
+{
+  char stmt_type = CUBRID_STMT_NONE;
+  const char *comment_start = strstr (stmt, "/*");
+  if (comment_start)
+    {
+      const char *comment_end = strstr (comment_start, "*/");
+      if (comment_end)
+	{
+	  char type_name[30] = { 0, };
+	  const char *dblink_start = strstr (comment_start, "DBLINK");
+	  if (dblink_start)
+	    {
+	      strncpy (type_name, dblink_start + strlen ("DBLINK"), comment_end - dblink_start - strlen ("DBLINK"));
+	      type_name[comment_end - dblink_start - strlen ("DBLINK")] = '\0';
+
+	      ut_trim (type_name);
+	      ut_tolower (type_name);
+
+	      stmt_type = get_stmt_type (type_name);
+	    }
+	}
+    }
+
+  if (stmt_type == CUBRID_STMT_NONE)
```

Comment:
get_stmt_type () returns **CUBRID_MAX_STMT_TYPE** if there is no string matching the input argument. Is it intended that it does not return CUBRID_STMT_NONE?

Reply:
Exception handling will be added when CUBRID_MAX_STMT_TYPE is returned.